app/histories_v2.go

```diff
@@ -39,6 +39,14 @@ func HistoriesV2Handler(app *App) func(c echo.Context) error {
 			topicMessages := mongoclient.GetMessagesV2(c, topic, from, limit, collection)
 			messages = append(messages, topicMessages...)
 		}
+
+		gameId := messages[0].GameId
```

Comment:
To avoid any panic error, check if the message slice has any message first. If not, you don't need to put any tag into the metric tags map.